book/chapters/chapter12/model_interpretation.qmd

```diff
@@ -7,26 +7,28 @@
 
-1. To gain global insights into a model, for example, to identify which features were the most important overall.
+1. To gain global insights into a model, for example, to identify which features were the most important overall or how the features act on the predictions.
 2. To improve the model if flaws are identified (in the data or model), for example, if the model depends on one feature unexpectedly.
 3. To understand and control individual predictions, for example, to identify how a given prediction may change if a feature is altered.
 4. To assess `r index('algorithmic fairness')`, for example, to inspect whether the model adversely affects certain subpopulations or individuals (see @sec-fairness).
 
 In this chapter, we will look at model-agnostic (i.e., can be applied to any model) `r index('interpretable machine learning', aside = TRUE)` (IML) methods that can be used to understand models `r index("post hoc")` (after they have been trained).
-We will focus on methods implemented in three packages that interface with `mlr3`: `r ref_pkg("iml")` (@sec-iml), `r ref_pkg("counterfactuals")` (@sec-counterfactuals), and `r ref_pkg("DALEX")` (@sec-dalex).
+We will focus on methods implemented in three R packages that nicely interface with `mlr3`: `r ref_pkg("iml")` (@sec-iml), `r ref_pkg("counterfactuals")` (@sec-counterfactuals), and `r ref_pkg("DALEX")` (@sec-dalex).
 
 `iml` and `DALEX` offer similar functionality but differ in design choices in that `iml` makes use of the `R6` class system whereas `DALEX` is based on the S3 class system.
-`counterfactuals` also uses `R6`.
-In contrast to `iml` and `counterfactuals`, `DALEX` focuses on comparing multiple predictive models, usually of different types, which is a model comparison-oriented approach often referred to as the `r index('Rashomon', lower = FALSE)` perspective.
+`counterfactuals` also uses the `R6` class system.
+In contrast to `iml` and `counterfactuals`, `DALEX` focuses on comparing multiple predictive models, usually of different types.
 We will only provide a brief overview of the methodology discussed below, we recommend @Molnar2022 as a comprehensive introductory book about IML.
 
-As a running example throughout this chapter, we will consider a gradient boosting machine (GBM) fit on half the features in the `"german_credit"` task:
+As a running example throughout this chapter, we will consider an gradient boosting machine (GBM) fit on half the features in the `"german_credit"` task.
+In practice, we would tune the hyperparameters of GBM as discussed in @sec-optimization and perform feature selection as discussed in @sec-feature-selection to select the most relevant features. 
+However, for the sake of simplicity, we opted to utilize an untuned GBM for the `"german_credit"` task, considering that it exhibited satisfactory performance even without fine-tuning, saving the reader valuable time and resources when running the example code below:
```

Comment:
saving the reader valuable time and resources when running the example code below: --> remove